### ISSUE TITLE
make sure `facing=` key is explicitly specified

### ISSUE THREAD

**Issue by cooljeanius** [knyghtmare/Start_of_the_War#2]
Sometimes characters will be facing odd directions; make sure to explicitly specify which way you actually want them to face

**Comment by knyghtmare** [comment]
> Sometimes characters will be facing odd directions; make sure to explicitly specify which way you actually want them to face

Good catch. There was a cutscene where the MC had his back turn on the king XD. Laughed my socks off. XD

**Comment by cooljeanius** [comment]
I think this might actually no longer be strictly necessary due to changes in upstream Wesnoth, but it's probably still worth doing anyways, just in case.
